src/biaplotter/artists.py

```diff
@@ -308,7 +321,6 @@ def overlay_colormap(self) -> BiaColormap:
     def overlay_colormap(self, value: Colormap):
         """Sets the overlay colormap for the scatter plot."""
         self._overlay_colormap = BiaColormap(value)
-        self.color_indices = self._color_indices
```

Comment:
Why was this line removed? If color_indices are not updated after a setting a new overly colormap, the effect of adding the new colormap is not seen until one explicit updates color_indices (check cell  16 of the scatter_artist_examle notebook).